ouroboros-consensus/test-storage/Test/Ouroboros/Storage/ChainDB/Model.hs

```diff
@@ -364,12 +361,7 @@ addBlock cfg blk m
   where
     secParam = protocolSecurityParam cfg
 
-    -- If we switch to PBFT for these tests, this case is not required anymore
-    -- TODO: We should not make assumptions about the underlying
-    -- ledger. We will fix this in
-    -- <https://github.com/input-output-hk/ouroboros-network/issues/1571>
```

Comment:
This special case is gone without #1571. Unverified: maybe we only needed it to not ignore the EBB, but we still don't adopt EBBs immediately when adding them, only when their successor is added? If that's true, then PBFT would change that, but then it is not that important to make that change.

Reply:
Hang on, didn't we verify this? And it still failed? What am I getting confused with?

Reply:
(This will have a minor conflict with my PR I guess.)